### PR TITLE
Add Disclaimer page

### PR DESCRIPTION
**Pull request by 409H**
[CH8841]

Note: I added the disclaimer copy from 2020 - will need a review from Jordan

### CODE REVIEW

File: src/assets/content/disclaimer.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+slug: '/disclaimer'
+date: '2021-01-23'
+title: 'Disclaimer'
+---
+
+**Be safe & secure:** [We highly recommend that you read our guide on How to Prevent Loss & Theft for some recommendations on how to be proactive about your security.](https://support.mycrypto.com/staying-safe/protecting-yourself-and-your-funds)
+
+**Always backup your keys:** MyCrypto.com & MyCrypto CX are not "web wallets". You do not create an account or give us your funds to hold onto. No data leaves your computer / your browser. We make it easy for you to create, save, and access your information and interact with the blockchain.
```
Comment (jspence425) [inline src/assets/content/disclaimer.md:9]: **Always backup your keys:** MyCrypto.com & the MyCrypto Desktop Application are not "web wallets". You do not create an account or give us your funds to hold onto. No data leaves your computer / your browser. We make it easy for you to create, save, and access your information and interact with the blockchain.

File: src/assets/content/disclaimer.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+**We are not responsible for any loss:** Ethereum, MyCrypto.com & MyCrypto CX, and some of the underlying Javascript libraries we use are under active development. While we have thoroughly tested & tens of thousands of wallets have been successfully created by people all over the globe, there is always the possibility something unexpected happens that causes your funds to be lost. Please do not invest more than you are willing to lose, and please be careful.
```
Comment (jspence425) [inline src/assets/content/disclaimer.md:11]: **We are not responsible for any loss:** Ethereum, MyCrypto.com, the MyCrypto Desktop Application, and some of the underlying Javascript libraries we use are under active development. While we have thoroughly tested our products and tens of thousands of wallets have been successfully created by people all over the globe, there is always the possibility something unexpected happens that causes your funds to be lost. Please do not invest more than you are willing to lose, and please be careful.

File: src/assets/content/disclaimer.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+**Default Network Endorsements:** MyCrypto, by default, supports all Ethereum-based blockchains ("networks") through the adding of custom nodes. To save time and provide a convenience factor to users, if an Ethereum-based network has enough perceived interest or users, MyCrypto will accept GitHub pull requests to add that network as one of the default supported networks. This will bypass the user's need to add that blockchain as a custom node in the MyCrypto network selector.
+
+If MyCrypto chooses to implement support for a specific network, it should be noted that MyCrypto does not assume the responsibility of managing that network.
+
+A network being added to MyCrypto's default supported networks list does not mean that MyCrypto has a partnership with said network, and is not an endorsement of said network.
```
Comment (jspence425) [inline src/assets/content/disclaimer.md:17]: mean > indicate

File: src/assets/content/disclaimer.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+**Translations of MyCrypto:** The community has done an amazing job translating MyCrypto into a variety of languages. However, MyCrypto can only verify the validity and accuracy of the information provided in English and, because of this, the English version of our website is the official text.
+
+**MIT License**
+Copyright (c) 2015-2017 MyEtherWallet LLC
+Copyright (c) 2019 MyCrypto, Inc.
```
Comment (jspence425) [inline src/assets/content/disclaimer.md:23]: **MIT License**
Copyright (c) 2015-2017 MyEtherWallet LLC
Copyright (c) 2019-2021 MyCrypto, Inc.

**Review by jspence425 — CHANGES_REQUESTED** [review]
left comments!

File: src/components/ContactUsBox.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 17px;
+  color: #424242;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 16px 0px;
+`;
```
Comment (GuillaumeRx) [inline src/components/ContactUsBox.tsx:39]: Please  convert this to Rebass :)

**Review by GuillaumeRx — CHANGES_REQUESTED** [review]

File: src/components/ContactUsBox.tsx (hunk 5 above)
Comment (409H) [reply src/components/ContactUsBox.tsx]: https://github.com/MyCryptoHQ/landing/pull/87/commits/2b0ac884b937c7921994b77deebec136e90785eb

File: src/components/ContactUsBox.tsx [hunk 6]
```diff
@@ -1,53 +1,26 @@
...
+        <Text
+          variant="smallBase"
+          mt="14px"
+          mb="14px"
+          lineHeight="17px"
+          fontSize="14px"
+        >
+          If you have any questions, comments, or suggestions about our
+          disclaimer, please contact MyCrypto support at{' '}
+          <a href="mailto:support@mycrypto.com">support@mycrypto.com</a>
```
Comment (GuillaumeRx) [inline src/components/ContactUsBox.tsx:20]: use the `Link` component please

**Review by GuillaumeRx — CHANGES_REQUESTED** [review]

File: src/components/ContactUsBox.tsx (hunk 6 above)
Comment (409H) [reply src/components/ContactUsBox.tsx]: https://github.com/MyCryptoHQ/landing/pull/87/commits/8094f981554fd0f6e4ffe8ce518e482065c4bff0

**Review by GuillaumeRx — APPROVED** [review]

**Pull request merged**